src/core/BufferGeometry.js

```diff
@@ -811,6 +851,60 @@ Object.assign( BufferGeometry.prototype, EventDispatcher.prototype, {
 
 	},
 
+	/**
+	 * Checks for duplicate vertices with hashmap.
+	 * Faces with duplicate vertices are rewired to point to the first instance.
+	 * Unused vertices are not removed.
+	 * @param  {THREE.BufferGeometry} geom Geometry to merge
+	 */
+	mergeVertices: function( geom ) {
```

Comment:
This function was for my computeCuspNormals function that I also wrote (but did not include), I will remove this here and file a separate PR later.